docs/learn/learn-staking-faq.md

```diff
@@ -79,13 +79,22 @@ min-intention-threshold (10 DOT) which is required to submit your intention to n
 supports 50,000 nominator intentions, but can reward at most 22500 nominators in any given era. If
 you nominate with, say 50 DOT, you are not guaranteed to be part of the top 22500 and hence you may
 not earn staking rewards. The entry barrier to be an electing nominator is now determined
-dynamically, based on market demand.
+dynamically, based on market demand. 
 
 The nominator intentions (up to 50,000) serve as a waitlist for the electing subset (22,500) which
 receives rewards in each era. Having more number of nominator intentions than the electing subset
 allows for the dynamic staking system to have some flexibility. For example, assume the top 1000
 nominators all unbond their stake, having that extra buffer of nomination intentions becomes useful.
 
+:::info If your account is in the last bag receiving staking rewards
+
+Within the context of a single bag, nodes are not sorted by their stake, but instead placed in insertion 
```

Comment:
can combine the statements related to insertion order to create clarity: 

`...but instead placed in insertion order where accounts can put themselves in front of ...`

Reply:
The sentence would read too long.